src/Popover/Popover.js

```diff
@@ -280,40 +318,38 @@ export default class Popover extends Component<Props, State> {
     }
   };
 
-  isClickOutsideComponent = (event: SyntheticEvent<>) => {
+  isEventOutsideComponent = (event: SyntheticEvent<>) => {
     /* eslint-disable react/no-find-dom-node */
     const { usePortal } = this.props;
     const node = findDOMNode(this);
     const popoverContentNode = findDOMNode(this.popoverContent);
+    /* $FlowFixMe */
```

Comment:
We don't need a FlowFixMe if we use the following code: 
```js
isEventOutsideComponent = (
    event: SyntheticEvent<> | SyntheticFocusEvent<>
  ) => {
    /* eslint-disable react/no-find-dom-node */
    const { usePortal } = this.props;
    const node = findDOMNode(this);
    const popoverContentNode = findDOMNode(this.popoverContent);
    const target =
      event.type === 'blur' && event.relatedTarget
        ? event.relatedTarget
        : event.target;
```

Reply:
Cool, will try tomorrow!